src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -264,7 +263,9 @@ def __init__(
         self.mount_target = mount_target if mount_target else "/home/container/workdir/"
         self.mount_source = mount_source if mount_source else MOUNT_SOURCE
         try:
-            self.host_port = port if port else random.randint(5000, 6000)
+            import secrets
+
+            self.host_port = port if port else secrets.randbelow(1000) + 5000
```

Comment:
Issue in the original code: the deduced port can be unavailable in the host. A better logic here would be to run a loop between the lower and upper limit till an availabe port is found.

Reply:
Right, updated now. Thank you.